test/e2e/app/app.go

```diff
@@ -27,6 +27,23 @@ type Application struct {
 	restoreChunks   [][]byte
 }
 
+type Config struct {
+	Dir              string
+	SnapshotInterval uint64
```

Comment:
What are the units here? Number of blocks I'm guessing?

Reply:
Yup